libs/blocks/icon-block/icon-block.js

```diff
@@ -22,19 +22,19 @@ function decorateLayout(el) {
     decorateBlockBg(el, children[0]);
   }
   const foreground = document.createElement('div');
```

Comment:
Do we need a new element here? I think we can likely just use the given dom and add a 'foreground' class to the children[1] node if (children.length > 1). If a new el is warranted, we should import and use `createTag()`. 
Maybe a thought for future `icon-block` updates.